docs/migration/v8-to-v9.md

```diff
@@ -234,6 +234,10 @@ Sentry.init({
 - The `addNormalizedRequestDataToEvent` method has been removed. Use `httpRequestToRequestData` instead and put the resulting object directly on `event.request`.
 - A `sampleRand` field on `PropagationContext` is now required. This is relevant if you used `scope.setPropagationContext(...)`
 
+### `@sentry/solidstart`
+
+- The `sentrySolidStartVite` plugin is no longer exported. Instead, wrap the SolidStart config with `withSentry` and provide Sentry options as the second parameter.
```

Comment:
If you want to be extra cool, you could add an example code snippet here of how that migration will look. Once we write this migration guide out we are gonna have more of those so we might already get started with that.